docs/README.md

```diff
@@ -9,5 +9,21 @@ It is possible to wrap a React element with [Overlay](https://github.com/vtex-ap
 ```
 <Overlay>
   <div>Hello World!</div>
-</Overlay> 
+</Overlay>
 ```
+
+## Props
+
+| Prop name    | Type                  | Description                                                  | Default value |
+| ------------ | --------------------- | ------------------------------------------------------------ | ------------- |
+| `alignment`  | `HorizontalAlignment` | Horizontal alignment                                         | `left`        |
```

Comment:
It may be better to consider its type as "string" and show the possible values.

Reply:
Done!